### DISCUSSION TITLE
PerlMagic with ActivePerl 5.28

### DISCUSSION THREAD

**Discussion by mbreeding** [ImageMagick/ImageMagick#2800] [Help]
I have been using PerlMagic with ActiveState Perl for many years in my WIndows Server environment.  It was originally installed using PPM wiht the Image-Magick.ppd created through when ImageMagick is installed on the machine.
Now that ActiveState has moved to its new platform for installing Perl modules, the PPM method is no longer possible.  The ActiveState platform offers PerlMagick as an module that can be included with a build, but it does not build.  My guess is that it does not have access to the specific libraries related to the ImageMagic installation.
I have tried manually copying the Magick.pm, Magick.dll and other files to ther respective locations with my Perl64 directories but this approach has not worked either.
Due to other considerations I cannot change to Strawberry Perl.
Is there a workaround for this scenario?

**Comment by urban-warrior** [comment]
Its been years since we last qualified PerlMagick against ActivateState Perl.  The ImageMagick developers currently do not have the time to investigate.  Hopefully one of ImageMagick users familiar with ActiveState Perl will be able offer advice.

**Reply by mbreeding** [reply]
Thanks for responding.  I mostly use PerlMagick to resize images.  I have found some other modules able to perform these limited tasks.
